src/client.js

```diff
@@ -3,17 +3,15 @@
  * Module dependencies.
  */
 
+import { Octokit } from '@octokit/rest';
 import { differenceBy, get, has } from 'lodash';
-import Github from '@octokit/rest';
 
 /**
  * GitHub configuration.
  */
 
 const config = {
```

Comment:
No need to have this `config` object anymore. Just create a `timeout` constant (or `defaultTimeout` based on the next comment).

Besides that, as a suggestion, it could just be a default value in case the constructor does not receive a specified timeout value. You would have something like this:

```js
constructor({ timeout, token, ...options }) {
    this.github = new Octokit({
        auth: token,
        request: { timeout: timeout || defaultTimeout },
        ...options
    });
}
```